crates/types/Cargo.toml

```diff
@@ -23,6 +23,7 @@ restate-errors = { workspace = true }
 restate-serde-util = { workspace = true }
 restate-test-util = { workspace = true, optional = true }
 restate-utoipa = { workspace = true }
+tonic = { workspace = true }
```

Comment:
We separate restate dependencies and other dependencies into two blocks. It would be great if you could add this dependency to the following block (sorted alphabetically).